src/modules/seventv/channel-emotes.js

```diff
@@ -55,13 +55,14 @@ class SevenTVChannelEmotes extends AbstractEmotes {
         for (const {
           id,
           name: code,
+          flags,
           data: {listed, animated, owner},
         } of emotes) {
           if (!listed) {
             continue;
           }
 
-          this.emotes.set(code, createEmote(id, code, animated, owner, category));
+          this.emotes.set(code, createEmote(id, code, animated, owner, category, flags));
```

Comment:
we also need to support the events below. i believe for those `visibility` has flag `1 << 7`.

we may want to consider passing a boolean instead of parsing flags in the `createEmote` utility

Reply:
```
event: ready
data: 7tv-event-sub.v1

event: update
data: {"channel":"vasp","emote_id":"60ae65b29627f9aff4fd8bef","name":"NOOOO","action":"ADD","actor":"Vasp","emote":{"name":"NOOOO","visibility":0,"mime":"image/webp","tags":[],"width":[34,68,102,136],"height":[32,64,96,128],"urls":[["1","https://cdn.7tv.app/emote/60ae65b29627f9aff4fd8bef/1x.webp"],["2","https://cdn.7tv.app/emote/60ae65b29627f9aff4fd8bef/2x.webp"],["3","https://cdn.7tv.app/emote/60ae65b29627f9aff4fd8bef/3x.webp"],["4","https://cdn.7tv.app/emote/60ae65b29627f9aff4fd8bef/4x.webp"]],"owner":{"id":"60ae49350e35477634486602","twitch_id":"238682742","display_name":"JustRogan","login":"justrogan"}}}

event: heartbeat
data: {}
```

Flags aren't provided it seems